template.rb

```diff
@@ -332,7 +328,13 @@ def print_announcements
       </div>
     </main>
   EOHTML
-  append_to_file "config/initializers/assets.rb", "Rails.application.config.assets.paths << Rails.root.join(\"node_modules\")"
+  append_to_file "config/initializers/assets.rb", <<~EOC
+    Rails.application.configure do
+      config.assets.paths << Rails.root.join("node_modules/@uswds/uswds/dist/img")
+      config.assets.paths << Rails.root.join("node_modules/@uswds/uswds/dist/fonts")
+      config.assets.excluded_paths << Rails.root.join("app/assets/stylesheets")
+    end
+  EOC
```

Comment:
[praise] I don't actually know the details but the increased specificity at least _seems_ nice.

Reply:
it's both nice, and entirely necessary given how propshaft uses that variable. After switching to propshaft but before updating these lines it was copying the entirety of `node_modules` into the `public/assets` folder, which led to running out of disk space and making every single file available to anyone browsing the site